drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -92,32 +96,30 @@ static u64_t fix_timestamp;
 
 static void print_satellite_stats(nrf_gnss_data_frame_t *pvt_data)
 {
-	u8_t  tracked          = 0;
-	u8_t  in_fix           = 0;
-	u8_t  unhealthy        = 0;
+	u8_t  n_tracked = 0;
+	u8_t  n_used = 0;
+	u8_t  n_unhealthy = 0;
 
 	for (int i = 0; i < NRF_GNSS_MAX_SATELLITES; ++i) {
-
-		if ((pvt_data->pvt.sv[i].sv > 0) &&
-		    (pvt_data->pvt.sv[i].sv < 33)) {
-
-			tracked++;
-
-			if (pvt_data->pvt.sv[i].flags &
-					NRF_GNSS_PVT_FLAG_FIX_VALID_BIT) {
-				in_fix++;
-			}
-
-			if (pvt_data->pvt.sv[i].flags &
-					NRF_GNSS_SV_FLAG_UNHEALTHY) {
-				unhealthy++;
-			}
+		u8_t sv = pvt_data->pvt.sv[i].sv;
+		bool used = (pvt_data->pvt.sv[i].flags &
+			     NRF_GNSS_SV_FLAG_USED_IN_FIX) ? true : false;
+		bool unhealthy = (pvt_data->pvt.sv[i].flags &
+				  NRF_GNSS_SV_FLAG_UNHEALTHY) ? true : false;
+
+		if ((sv > 0) && (sv < 33)) {
```

Comment:
If we improve readability here, could we also replace the magic numbers with symbols? Or at least add a comment on what 0 and 33 mean in this case.